x-pack/packages/security/plugin_types_public/src/user_profile/user_profile_api_client.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type {
+  UserProfileData,
+  AuthenticatedUser,
+  UserProfileWithSecurity,
+  UserProfile,
+} from '@kbn/security-plugin-types-common';
+import { Observable } from 'rxjs';
```

Comment:
nit: import type

Reply:
Ah, good catch! I'll fix.

Reply:
Handled in https://github.com/elastic/kibana/pull/171886/commits/4e04f9db4123ee9688adc5d3d2e53b6087faa89d